app/models/post.rb

```diff
@@ -31,15 +31,19 @@ class Post < ApplicationRecord
   has_many :indexes, inverse_of: :posts, through: :index_posts, dependent: :destroy
   has_many :index_sections, inverse_of: :posts, through: :index_posts, dependent: :destroy
 
-  attr_accessor :is_import
+  has_one :written, -> { where(reply_order: 0) }, class_name: 'Reply', inverse_of: :post
+
+  attr_accessor :is_import, :skip_written
```

Comment:
Can we put a TODO to delete skip_written after the migration, or is that stored in a checklist somewhere?